.github/workflows/lambda_staging.yml

```diff
@@ -54,6 +54,48 @@ jobs:
             --function-name ${{ matrix.image }} \
             --image-uri $REGISTRY/${{ matrix.image }}:${GITHUB_SHA::7} > /dev/null 2>&1
 
+      - name: Set Lambda New Relic Toggle and Env Var Names
+        run: |
+          set -euo pipefail
+
+          PARAM_NAME="ENVIRONMENT_VARIABLES"
+          RAW=$(aws ssm get-parameter \
+            --name "$PARAM_NAME" \
+            --with-decryption \
+            --query 'Parameter.Value' \
+            --output text)
+
+          # Extract NEW_RELIC_ENABLED (default to false if absent)
+          NEW_RELIC_ENABLED=$(printf '%s\n' "$RAW" | grep '^NEW_RELIC_ENABLED=' | head -n1 | cut -d '=' -f2- || true)
+          if [ -z "${NEW_RELIC_ENABLED}" ]; then
+            NEW_RELIC_ENABLED=false
+          fi
+
+          # Build CONFIG_KEYS: list of all keys (names only)
+          CONFIG_KEYS=$(printf '%s\n' "$RAW" | grep -v '^[[:space:]]*$' | grep -v '^#' | cut -d '=' -f 1 | paste -sd ',' -)
+
+          # Truncate CONFIG_KEYS if extremely long to avoid hitting limits
+          MAX_LEN=3500
+          if [ ${#CONFIG_KEYS} -gt $MAX_LEN ]; then
+            CONFIG_KEYS_TRUNCATED="${CONFIG_KEYS:0:$MAX_LEN}"
+            echo "CONFIG_KEYS truncated from ${#CONFIG_KEYS} to ${#CONFIG_KEYS_TRUNCATED} characters" >&2
+            CONFIG_KEYS="$CONFIG_KEYS_TRUNCATED"
+          fi
+
+          # Overwrite entire static environment: only keep these two keys
+          jq -n \
+            --arg nr "$NEW_RELIC_ENABLED" \
+            --arg keys "$CONFIG_KEYS" \
+            '{NEW_RELIC_ENABLED:$nr, CONFIG_KEYS:$keys}' > merged_env.json
```

Comment:
Why do we want to keep `NEW_RELIC_ENABLED`? and not have it part of CONFIG_KEYS listed like the others?